content/patterns/radio/examples/radio.html

```diff
@@ -163,25 +159,25 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
               <td><code>div</code></td>
               <td>
                 <ul>
-                  <li>Identifies the <code>div</code> element as a container for a group of <code>radio</code> buttons.</li>
-                  <li>Is not focusable because focus is managed using a roving tabindex strategy as described below.</li>
+                  <li><code>div</code> 要素を <code>radio</code> ボタンのグループのコンテナとして特定。</li>
+                  <li>以下で説明するように、フォーカスは「tabindex の移動 (roving tabindex)」戦略を使用して管理されるため、フォーカス可能ではない。</li>
                 </ul>
               </td>
             </tr>
             <tr data-test-id="radiogroup-aria-labelledby">
               <td></td>
               <th scope="row"><code>aria-labelledby="[IDREF]"</code></th>
               <td><code>div</code></td>
-              <td>Refers to the element that contains the label of the radio group.</td>
+              <td>ラジオグループのラベルを含む要素を参照。</td>
```

Comment:
体言止めはやりすぎで、言い切りの形にするくらいがちょうどいいでしょうか？

```suggestion
              <td>ラジオグループのラベルを含む要素を参照する。</td>
```